src/main/java/icbm/classic/content/blast/threaded/BlastAntimatter.java

```diff
@@ -24,11 +25,24 @@
 {
     private final IBlockState replaceState = Blocks.AIR.getDefaultState();
 
+    // CubicChunks support
+    int minWorldHeight;
+    int maxWorldHeight;
+
     @Override
     public boolean setupBlast()
     {
         super.setupBlast();
         ICBMSounds.ANTIMATTER.play(world, this.location.x(), this.location.y(), this.location.z(), 7F, (float) (this.world().rand.nextFloat() * 0.1 + 0.9F), true);
+
+        if (CubicChunks.INSTANCE.isPresent()) {
+            this.minWorldHeight = CubicChunks.INSTANCE.getMinHeight(world);
```

Comment:
Might be better to add a common helper with supplier functions. This way we disconnect the need to call CubicChunk class. As this makes it harder then needed to unit test the code.